app/models/onboarding_form_service.rb

```diff
@@ -132,7 +138,7 @@ def delete_config_files(files)
 
     private
 
-    def value_or_default(value, default = '-')
+    def value_or_default(value, default = '-- Not Provided --')
```

Comment:
Having just the dash was interpreted in zendesk as markdown for the preceding line being a heading. This solves the problem.